docs/AboutUs.md

```diff
@@ -5,55 +5,17 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `e0417539@u.nus.edu`
 
 ## Project team
 
-### John Doe
+### Xing Yu
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/xnoobftw.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
```

Comment:
You can leave the template inside first, then at the same time can also put your github and portfolio link inside. Think we should try to use their format as close as possible, what do you think?